README_validations.md

```diff
@@ -0,0 +1,116 @@
+
+## What are the OSCAL models?
+
+OSCAL comes in several different forms, with data models in JSON, XML, and YAML. Please review [this page with the status of all the models as organized into layers](https://pages.nist.gov/OSCAL/concepts/layer/#status-of-the-oscal-layers) for information about each model and the status of its respective schemas.
+
+## OSCAL "complete" schema
+
+Additionally, there are "complete" XML and JSON schemas, which can validate any kind of OSCAL. It is larger and more complex than the model-specific schemas and may not be as efficient, but it can be convenient to use one schema file as an entrypoint to validate one or more OSCAL documents without knowing the particular model in advance, whether it be a `component-definition`, `system-security-plan`, `assessment-plan`, et cetera.
+
+## How do I apply a schema to a document to determine validity?
+
+Schemas are built using standard schema languages that are supported by commodity tools. These tools can all apply the same schemas to the same documents, and give the same results.
+
+For OSCAL XML, this schema syntax is [XML Schema Definition (XSD)](https://www.w3.org/TR/xmlschema-1/). For OSCAL JSON, the schema syntax is [JSON Schema](https://json-schema.org/). Many freely available, open-source, and commercial implementations are available. These schema standards are also built in to many common application frameworks and development environments.
+
+Each tool will have its own way to provide for validation. Bindings or configurations to link documents to schemas can be maintained either inside or outside the documents themselves; many tools offer several ways to do this.
+
+Some tools support self-contained validation functionality without full-fledged application frameworks or development environments. Validation against a XML schema in XSD syntax can be done in a single command, for example, with a tool like [`xmllint`](http://xmlsoft.org/xmllint.html) (where `oscal-schema.xsd` is the appropriate schema and `oscal.xml` is an example OSCAL document).
+
+```
+$ xmllint oscal.xml --schema oscal-schema.xsd --noout
+```
+
+Similarly, using the [`ajv-cli`](https://github.com/ajv-validator/ajv-cli), you can validate a JSON document like so:
+
+```
+# Install schema type library in NodeJS runtime to avoid runtime errors
+# and run the ajv command line tool after installation.
+npm install -g ajv-formats && npx ajv validate -s oscal-schema.json -d oscal.json -c ajv-formats
+```
+
+## What does success look like?
+
+If you validate an OSCAL catalog with its schema (a local copy) as above (using `xmllint`), it will return results to the standard output (STDOUT) stream like so:
+
+```
+$ xmllint oscal.xml --schema oscal-schema.xsd --noout
+oscal.xml validates
+```
```

Comment:
Duplicated from above. Why?

Reply:
That might have been a mistake, checking!